ui/app/components/get-credentials-card.hbs

```diff
@@ -3,24 +3,28 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<form {{on "submit" this.transitionToCredential}} class="selectable-card is-rounded no-flex">
```

Comment:
Just wrapped this component in `<Hds::Card>` to remove the styling classes

Reply:
before/after
<img width="918" alt="Screenshot 2023-10-20 at 11 50 29 AM" src="https://github.com/hashicorp/vault/assets/68122737/97eed740-d1ac-4c5f-8770-3d29627bb4f6">